packages/lesswrong/lib/routes.ts

```diff
@@ -929,6 +929,19 @@ const eaLwAfForumSpecificRoutes = forumSelect<Route[]>({
       path: '/meta',
       redirect: () => `/tag/site-meta`,
     },
+    {
+      name: 'bestOfLessWrongAdmin',
+      path: '/bestoflesswrongadmin',
+      // this is a hack because it's annoying to fetch ReviewWinnerArt by review year
+      // instead we fetch by createdAt date for the art, which is generally 2 years after review
```

Comment:
I don't quite understand what this comment has to do with the redirect (which seems to just be defaulting to some year offset)

Reply:
I updated the comment, hopefully making it more clear that its about why the particular year-offset was chosen